src/MediaManager/MediaManagerListRequest.php

```diff
@@ -120,13 +128,20 @@ public function getPageSize()
 
     /**
      * @param int $pageSize
+     *
+     * @return void
      */
     public function setPageSize($pageSize)
     {
         $this->pageSize = $pageSize;
     }
 
     /**
+     * @FIXME `$this->sortColumn` is annotated as `SortColumnInterface|null` and `MediaItemDataAccess::getOrderBy()` uses it
+     *         as such. But `MediaManagerListRequestFactory` fills it with `string|null` from `MediaManagerListState` which
+     *         in turn is set to a string directly from the request in `MediaManagerListStateFromRequestService`.
+     * @psalm-suppress InvalidReturnType, InvalidReturnStatement
+     *
      * @return string|null
      */
     public function getSortColumn()
```

Comment:
As far as I can tell, `sortColumn` can never actually be `SortColumnInterface` but is expected to be by consumers of this class